bundles/bundles.go

```diff
@@ -140,6 +171,10 @@ func NewBundleInstaller(bc BundleConfig) (BundleInstaller, error) {
 	case "container":
 		return &OCIImageExtractor{}, nil
 	case "run":
+		if bc.LocalFile {
+			return &LocalRunner{}, nil
+		}
+
 		return &OCIImageRunner{}, nil
 	case "package":
 		return &LuetInstaller{}, nil
```

Comment:
This installer is still shelling out to `luet`: https://github.com/kairos-io/kairos-sdk/blob/f73090dca2ae3ddc4023215836ebafd6132245f8/bundles/bundles.go#L207

Didn't we say we removed `luet` binary from our dependencies, thus we don't want to introduce it again?

Reply:
I'll see if I can do the TODO above. The `luet` type seems to be the same thing as the oci extractor above.

Reply:
Done (did grub-config intead)

Reply:
Hmm, changing this might introduce bugs. I'd rather fix what's broken and keep the rest intact. Let's consider changing in another PR if needed.